ts/Stock/Navigator/Navigator.ts

```diff
@@ -2036,6 +2036,19 @@ class Navigator {
                                 0
                         ) + navigator.navigatorOptions.margin;
                 }
+            ),
+            addEvent(
+                Navigator,
+                'setRange',
+                function (this: Navigator, e: SetRangeEvent):void {
+                    this.chart.xAxis[0].setExtremes(
+                        e.min,
+                        e.max,
+                        e.redraw,
+                        e.animation,
```

Comment:
If the `e.animation` is undefined, that will result in triggering the animation on setExtremes, which does not look good and decreases the performance. (This does not happen in normal navigator).